applications/adv_networking_bench/cpp/default_bench_op_rx.h

```diff
@@ -155,6 +155,11 @@ class AdvNetworkingBenchDefaultRxOp : public Operator {
                          "Header size",
                          "Header size on each packet from L4 and below",
                          42);
+    spec.param<bool>(reorder_kernel_,
```

Comment:
why is this needed?

Reply:
The reorder kernel shouldn't run when we're in CPU or HDS mode in some cases or it crashes because of buffer alignment. This was one of them

Reply:
per discussions, could we see that the packet length isn't divisible by 4 and just have that `reorder_kernel` turned off (+ a warning the first time) without user explicit set?